lib/Command/Configure.php

```diff
@@ -313,6 +316,29 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setUser($username);
 				$providerConfig->setApiKey($apiKey);
 
+				break;
+			case 'plivo':
+				$config->setProvider($provider);
+				/** @var PlivoConfig $providerConfig */
+				$providerConfig = $config->getProvider()->getConfig();
+				
+				$authIdQuestion = new Question('Please enter your plivo authentication id (Auth ID): ');
+				$authId = $helper->ask($input, $output, $authIdQuestion);
+				
+				$authTokenQuestion = new Question('Please enter your plivo authentication token (Auth Token): ');
+				$authToken = $helper->ask($input, $output, $authTokenQuestion);
+				
+				$srcNumberQuestion = new Question("Please enter your plivo phone number (in E.164 format '+12345678901'): ");
+				$srcNumber = $helper->ask($input, $output, $srcNumberQuestion);
+				
+				$callbackUrlQuestion = new Question('Please enter your plivo callback url: ');
+				$callbackUrl = $helper->ask($input, $output, $callbackUrlQuestion);
+				
+				$providerConfig->setValue($providerConfig::AUTH_ID_KEY,$authId);
```

Comment:
```suggestion
				$providerConfig->setValue(PlivoConfig::AUTH_ID_KEY,$authId);
```

you access a constant of the class here, not of the object. strictly speaking php might allow both, but I find it cleaner if the class is used.

Reply:
I agree.